classes/plugins/PubIdPlugin.inc.php

```diff
@@ -226,6 +226,26 @@ function getFormFieldNames() {
 	}
 
 	/**
+	 * Get the checkbox form field name that is used to define
+	 * if a pub object should be excluded from assigning the pub id to it.
+	 * @return string
+	 */
+	function getExcludeFormFieldName() {
+		assert(false); // Should be overridden
```

Comment:
When this gets forward-ported to master, note that we're now using `abstract function` rather than `assert(false);` -- it's cleaner.